pkg/cmd/attestation/verification/extensions.go

```diff
@@ -0,0 +1,28 @@
+package verification
+
+import (
+	"fmt"
+	"strings"
+)
+
+func VerifyCertExtensions(results []*AttestationProcessingResult, owner string, repo string) error {
+	for _, attestation := range results {
+		// TODO: handle proxima prefix
+		expectedSourceRepositoryOwnerURI := fmt.Sprintf("https://github.com/%s", owner)
+		sourceRepositoryOwnerURI := attestation.VerificationResult.Signature.Certificate.Extensions.SourceRepositoryOwnerURI
+		if sourceRepositoryOwnerURI != "" && !strings.EqualFold(expectedSourceRepositoryOwnerURI, sourceRepositoryOwnerURI) {
```

Comment:
If `sourceRepositoryOwnerURI != ""`, then the extension is not set in the certificate, which I think should be an error instead of succeeding?

In sigstore-go, we check [actualFieldVal.IsValid()](https://github.com/sigstore/sigstore-go/blob/main/pkg/fulcio/certificate/summarize.go#L81), but I believe that will return `true` for an empty string.

Reply:
make sense. remove the logic of checking sourceRepositoryOwnerURI is empty.